libmscore/glissando.cpp

```diff
@@ -300,7 +300,7 @@ void Glissando::layout()
       // interpolate y-coord of intermediate points across total width and height
       qreal xCurr = 0.0;
       qreal yCurr;
-      for (int i = 0; i < int(spannerSegments().size()-1); i++) {
+      for (unsigned i = 0; i + 1 < spannerSegments().size(); i++) {
```

Comment:
Nope, the expression before was clear and is used often to express zero-based counting. The new version is absolutely unclear to a reader. Could you please revert the "-1" back?

Reply:
Not the `size_t` btw? Why?

Reply:
because it leads to other compiler warnings